api/src/main/java/net/md_5/bungee/command/PlayerCommand.java

```diff
@@ -31,20 +28,9 @@ public PlayerCommand(String name, String permission, String... aliases)
     public Iterable<String> onTabComplete(CommandSender sender, String[] args)
     {
         final String lastArg = ( args.length > 0 ) ? args[args.length - 1].toLowerCase( Locale.ROOT ) : "";
-        return Iterables.transform( Iterables.filter( ProxyServer.getInstance().getPlayers(), new Predicate<ProxiedPlayer>()
-        {
-            @Override
-            public boolean apply(ProxiedPlayer player)
-            {
-                return player.getName().toLowerCase( Locale.ROOT ).startsWith( lastArg );
-            }
-        } ), new Function<ProxiedPlayer, String>()
-        {
-            @Override
-            public String apply(ProxiedPlayer player)
-            {
-                return player.getName();
-            }
-        } );
+        return ProxyServer.getInstance().getPlayers().stream()
+            .map( player -> player.getName().toLowerCase( Locale.ROOT ) )
+            .filter( name -> name.startsWith( lastArg ) )
```

Comment:
The toLowerCase should be used in .filter, or the player names will be suggested in lower case (this is not what the original code do)
```suggestion
            .map( player -> player.getName() )
            .filter( name -> name.toLowerCase( Locale.ROOT ).startsWith( lastArg ) )
```